sdk/identity/azure-identity/tests/test_default_async.py

```diff
@@ -70,3 +72,32 @@
     # all credentials not representing managed identities should use a specified authority or default to public cloud
     await exercise_credentials("authority.com")
     await exercise_credentials(None, KnownAuthorities.AZURE_PUBLIC_CLOUD)
+
+
+def test_exclude_options():
+    def assert_credentials_not_present(chain, *credential_classes):
+        actual = {c.__class__ for c in chain.credentials}
+        assert len(actual)
+
+        # no unexpected credential is in the chain
+        excluded = set(credential_classes)
```

Comment:
Silly nit:  Is there a reason this is "credential_classes" and not "excluded_credential_classes" to be more precise.  (Took me a second of staring to make sure the logic in the len(&) was proper given the naming)

Reply:
No particular reason. Have renamed.